### DISCUSSION TITLE
Editing config

### DISCUSSION THREAD

**Discussion by PhantomKing50** [Scythe-Anticheat/Scythe-Anticheat#114] [Help and Support] [answered]
hiya so ive found the config file but i cant seem to edit it, could you tell me how?

**Comment by ExtraBytez** [comment]
It's in js and it's really simple turn the mcpack or Mcaddon into .zip then extract it then find the js files and open them in vocoder notepad text edit Kate etc then change it to your hearts desire then add the files back to a zip file and turn that zip into mcpack mcaddon etc del the main pack from your game and import the customized one

**Comment by ExtraBytez** [comment, marked as answer]
> It's in js and it's really simple turn the mcpack or Mcaddon into .zip then extract it then find the js files and open them in vocoder notepad text edit Kate etc then change it to your hearts desire then add the files back to a zip file and turn that zip into mcpack mcaddon etc del the main pack from your game and import the customized one

Should be marked as real answer as people will come back here
